pages/experiments.js

```diff
@@ -27,23 +27,34 @@ export default function experiments(props) {
   const displayExperiments = filteredExperiments.map((experiment) => (
     <li key={experiment.id} className="flex items-stretch">
       <Experiment
-        title={experiment.title}
-        tag={experiment.tag}
-        tagLabel={t(experiment.tag)}
-        description={experiment.description}
+        title={
+          props.locale === "en"
```

Comment:
I would say make en the default probably

Reply:
Nvm I think I see what you mean